content/04-guides/01-database-workflows/08-sql-views.mdx

```diff
@@ -0,0 +1,378 @@
+
+   ```
+   npm init -y
+   npm install prisma --save-dev
+   ```
+
+2. Create a new file called `schema.prisma` and add the following code to it:
+
+   ```prisma
+   datasource postgresql {
+       provider = "postgresql"
+       url      = env("DATABASE_URL")
+   }
+   ```
+
+3. In order to introspect your database, you need to tell Prisma how to connect to it. You do so by configuring a `datasource` in your Prisma schema. Create a new file called `.env` and set your database connection URL as the `DATABASE_URL` environment variable:
+
+   ```
+   DATABASE_URL=postgresql://__USER__:__PASSWORD__@__HOST__:__PORT__/CascadingDeletesDemo
+   ```
+
+   In the above code snippet, you need to replace the uppercase placeholders with your own connection details. For example, if your database is running locally it could look like this:
+
+   ```
+   DATABASE_URL=postgresql://janedoe:mypassword@localhost:5432/CascadingDeletesDemo
+   ```
+
+   The database connection URL is set via an environment variable. The Prisma CLI automatically supports the [`dotenv`](https://github.com/motdotla/dotenv) format which automatically picks up environment variables defined in a file called `.env`.
+
+4. With both the `schema.prisma` and `.env` files in place, run Prisma's introspection with the following command:
+
+   ```
+   npx prisma introspect
+   ```
+
+   This command introspects your database and for each table adds a Prisma model to the Prisma schema:
+
+   ```prisma
+    datasource postgresql {
+      provider = "postgresql"
+      url      = env("DATABASE_URL")
+    }
+
+    model Post {
+      authorId  Int?
+      content   String?
+      id        Int     @default(autoincrement()) @id
+      published Boolean @default(false)
+      title     String
+      User      User?   @relation(fields: [authorId], references: [id])
+    }
+
+    model User {
+      email String  @unique
+      id    Int     @default(autoincrement()) @id
+      name  String?
+      Post  Post[]
+    }
+   ```
+
+## 5. Manually add the Drafts view to the Prisma schema
+
+You must manually add views to the Prisma schema.
+
+1. Add a `Draft` model to the schema as shown:
+
+   > **Note**: The name of your view is case sensitive - if you created a view named `draft` in the database, you must create a model named `draft` in the Prisma schema.
+
+   ```prisma
+     datasource postgresql {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+
+   model Post {
+     authorId  Int?
+     content   String?
+     id        Int     @default(autoincrement()) @id
+     published Boolean @default(false)
+     title     String
+     User      User?   @relation(fields: [authorId], references: [id])
+   }
+
+   model User {
+     email String  @unique
+     id    Int     @default(autoincrement()) @id
+     name  String?
+     Post  Post[]
+   }
+
+   + model Draft {
+   +   title     String
+   +   id        Int     @unique
+   +   email     String
+   +   published Boolean
+   + }
```

Comment:
maybe better for format whole code block as `diff`? there will be no prisma highlighting in the new model anyway, and the other one is fully highlighted above I think.

Reply:
+1 🙂

Reply:
I would make this change, but the `diff` block disagrees:

![image](https://user-images.githubusercontent.com/849648/85310058-cf66fc00-b4b3-11ea-98c8-8e4b7c670c44.png)